app/routes/task_routes.py

```diff
@@ -0,0 +1,115 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from ..models.task import Task
+import datetime
+import os
+import requests
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
```

Comment:
Since `validate_model` appears in both task and goal routes, it's a good candidate for refactoring into its own file and importing as needed!